frontend/.coderabbit.yaml

```
@@ -0,0 +1,9 @@
+language: "ko-KR";
+
+reviews: profile: "chill";
+request_changes_workflow: false;
+high_level_summary: true;
+poem: false;
+review_status: true;
+auto_review: enabled: true;
+drafts: false;
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)